tmt/steps/provision/__init__.py

```diff
@@ -1365,6 +1373,23 @@ def is_ready(self) -> bool:
         # Enough for now, ssh connection can be created later
         return self.primary_address is not None
 
+    def setup(self) -> None:
+        if self.is_dry_run:
+            return
+        if not self.facts.is_superuser and self.become:
+            assert self.facts.package_manager is not None
+            self.execute(
+                Command(
+                    'sudo',
+                    f'{self.facts.package_manager.value}',
+                    'install',
+                    '-y',
+                    'acl'))
+            workdir_root = effective_workdir_root()
+            acl_setup_script = ShellScript(f'mkdir -p {workdir_root}')
+            acl_setup_script += ShellScript(f'setfacl -d -m o:rX {workdir_root}')
```

Comment:
`ShellScript` supports multiline strings, i.e. the following might be more readable:

```python
self.execute(ShellScript(
    f"""
    mkdir -p {workdir_root};
    setfacl -d -m o:rX {workdir_root}
    """))
```